backend/test/postman/starting-api.postman_collection.json

```diff
@@ -605,8 +605,7 @@
 							"",
 							"pm.test(\"JSON body is valid\", function () {",
 							"    var jsonData = pm.response.json();",
-							"    pm.expect(jsonData.message).to.eql('OK');",
-							"    pm.expect(jsonData.release).to.include(pm.environment.get(\"releaseVer\"));",
+							"    pm.expect(jsonData.status).to.eql('UP');",
```

Comment:
Since we also deploy a database, could we add a database check here as well, checking its status (`components.db.status` should be `UP`), and the validation query?

Reply:
Yup. We can add.